internal/checker/terraform/rules/aws/aws_redshiftserverless_namespace_redshiftserverless_namespace_encrypted_by_kms_utilizing_cmk.rego

```diff
@@ -0,0 +1,54 @@
+# METADATA
+# title: "A customer managed Key (CMK) should be used to encrypt the Redshift Serverless namespace"
+# description: "This quick test verifies that the Redshift Serverless namespace encrypts its contents with AWS key management - KMS."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/fsx_ontap_file_system
```

Comment:
@RamUseMemory1999 kindly verify if the link for related sources matched the resource